ugs-platform/ugs-platform-ugscore/src/main/java/com/willwinder/ugs/nbp/core/toolbars/Bundle.properties

```diff
@@ -0,0 +1,4 @@
+mainWindow.swing.firmware.toolbarTitle = Firmware selection toolbar
```

Comment:
These can be put in `LocalizingService.java` to statically load the translations from the properties files instead of having them localized in two places.